csfieldguide/static/interactives/viola-jones-face-detector/js/viola-jones.js

```diff
@@ -255,14 +264,46 @@ function loadImageDialog(input) {
       loadResizeImage();
     }
     reader.readAsDataURL(input.files[0]);
+    $("label[for='viola-jones-image-input']").text(input.files[0].name);
+    // reset position and validation border of haar boxes
+    $('#haar1').css('transform', 'translate(0, 0)').removeClass('valid');
+    $('#haar1').attr('data-x', 0);
```

Comment:
I spent so long trying to get this to work argh, much better style

Reply:
I appreciate you helping me out!!